src/lib.js

```diff
@@ -458,6 +458,11 @@ let lib = {
         // --devicetypeid is a string in the form "devicetype, runtime_version" (optional: runtime_version)
         let device = getDeviceFromDeviceTypeId(devicetypeid)
 
+        // log device information
+        console.log(util.format('device.name: %s', device.name))
```

Comment:
Do you think we should try to make these a bit more user friendlier since other tools also uses `ios-sim`?

For example: 'device.name: %s' become 'Device Name: %s'?

Reply:
Same here, just following more or less what was already part of the codebase - https://github.com/ios-control/ios-sim/blob/2e8cb284795bb3a656b6db59590c63f726e3e608/src/lib.js#L472. But the string really doesn't matter in this PR, it's about that this information is returned at all.

Reply:
👍